nb/api/size.md

```diff
@@ -0,0 +1,103 @@
+### size()
+Parameter|Default|Type
+---|---|---
+op|size => shape => size|Operates on the size.
+
+Analyzes the shape and calls op with an object with:
```

Comment:
Does .boundingBox() still exist? I kind of liked having the information returned like thisThingsSize = a.size();

Reply:
const foo = a.size(noOp()) should get you that, I think.

a.bb() exists and probably should have a long name like a.boundingBox()

Reply:
Nice, that works for me